src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static Random random = new Random();
+    private static final String[] COLORS = new String[] {
+            "red", "blue", "white", "brown", "green", "pink", "yellow", "orange"
+    };
+
+    public static String colorFigures() {
```

Comment:
same - please don't create static methods - they are in general bad practice
also, let's use verbs for method names
```suggestion
    public String getRandomColor() {
```